apps/official-journal-web/src/components/select/OJOISelect.tsx

```diff
@@ -1,20 +1,54 @@
-import { Select, SkeletonLoader } from '@island.is/island-ui/core'
+import {
+  Box,
+  BoxProps,
+  Select,
+  SkeletonLoader,
+} from '@island.is/island-ui/core'
 
-type Props<T> = React.ComponentProps<typeof Select<T>>
+import { OJOIInput } from './OJOIInput'
+
+type Props<T> = React.ComponentProps<typeof Select<T>> & {
+  width?: BoxProps['width']
+  isValidating?: boolean
+}
 
 export const OJOISelect = <T,>({
   filterConfig,
   isLoading,
+  isValidating,
+  width,
   ...rest
 }: Props<T>) => {
-  return isLoading ? (
-    <SkeletonLoader height={64} borderRadius="standard" />
-  ) : (
-    <Select
-      size="sm"
-      backgroundColor="blue"
-      {...rest}
-      filterConfig={{ matchFrom: 'start' }}
-    />
+  let valueString = ''
+  if (typeof rest.value === 'string') {
+    valueString = rest.value
+  }
+
+  if (rest.value && 'label' in rest.value) {
+    valueString = rest.value.label as string
+  }
```

Comment:
smá ruglandi að lesa, fattaði ekki fyrst að `rest.value` gæti verið object og ef svo, yfirskrifar úr fyrra if

líka bæta við tjékki á hvort strengur og þá sleppa `as`

```suggestion
  if (typeof rest.value === 'string') {
    valueString = rest.value
  } else if (rest.value && 'label' in rest.value && typeof rest.value.label === 'string') {
    valueString = rest.value.label
  }
```